### ISSUE TITLE
Some Features That The Software Needs

### ISSUE THREAD

**Issue by freewarecamp** [friction2d/friction#66]
(Previously Created on enve)

Thanks For The Software! I Really Hope That You are fine now. I just want to say some features that will help us working with friction.

1. Snapping
2. Grids
3. Easy Ease Animation (Graph) Shortcut
4. Speed Graph
5. Puppet Pin Effect
6. Camera Layer

**Comment by axiomgraph** [comment]
why do you need a speed graph

**Comment by freewarecamp** [comment]
Currently we have a value graph editor and its fine the way it is. But we have to make sure that x and y values are same while making changes in the graph. Whereas speed graph have only one curve to edit multiple values. I hope you get it.

**Comment by freewarecamp** [comment]
Yes Mr. Rodlie I respect your comment. A stable application is more important than having new features. That being said, I look forward to version 1.0 to see updates on friction in the future :)

**Comment by prokoudine** [comment]
When you request snapping, I hope you also mean on-canvas snapping and equal distribution aka "smart guides" like what you get in Adobe Illustrator, Figma, Penpot, Inkscape, and now even GIMP :)
